src/tango/tempo/fd_tempo.c

```diff
@@ -100,50 +100,66 @@ fd_tempo_tickcount_model( double * opt_tau ) {
   return t0;
 }
 
+static double mu;
+static double sigma;
+static int explicit_set;
+
+void
+fd_tempo_private_set_tick_per_ns( double _mu,
```

Comment:
I don't think this needs to be `private`. I'd be fine with it public, though make sure to document it.